code/framework/api/src/main/java/io/cattle/platform/api/utils/ApiUtils.java

```diff
@@ -186,7 +187,7 @@ public static String getSchemaIdForDisplay(SchemaFactory factory, Object obj) {
         if ( schema.getChildren().size() > 0 ) {
             String kind = ObjectUtils.getKind(obj);
             Schema kindSchema = factory.getSchema(kind);
-            if ( kindSchema != null ) {
+            if ( kindSchema != null && schema.getChildren().contains(kindSchema.getType()) ) {
```

Comment:
This issue here is that the code should get `getId()` and not `getType()` so

    schema.getChildren().contains(kindSchema.getId())